### ISSUE TITLE
Build fails in OBS

### ISSUE THREAD

**Issue by brejoc** [pop-os/gtk-theme#119]
I know, this is not really you target system, but I'm trying to build this theme for openSUSE Leap 42.3 and Tumbleweed on [OBS](http://build.opensuse.org/). Building and installing the theme on a native Leap 42.3 installation works like a charm. Doing the `make` and `make install` allows me to select and use the theme without any major issues.

But now I'm trying to build a package with the help of OBS. In the [spec-file](https://build.opensuse.org/package/view_file/home:brejoc:pop_os/pop-os-gtk-theme/pop-os-gtk-theme.spec?expand=1) I'm basically doing the same combination of `make` and `make install`. But here I'm getting this [error message](https://build.opensuse.org/package/live_build_log/home:brejoc:pop_os/pop-os-gtk-theme/openSUSE_Leap_42.3/x86_64):

```
ERROR: link target doesn't exist (neither in build root nor in installed system):
  /usr/share/themes/Pop-dark/gnome-shell/no-events.svg -> /usr/share/themes/Pop-dark/gnome-shell/assets/no-events.svg
Add the package providing the target to neededforbuild and Requires
ERROR: link target doesn't exist (neither in build root nor in installed system):
  /usr/share/themes/Pop-dark/gnome-shell/no-notifications.svg -> /usr/share/themes/Pop-dark/gnome-shell/assets/no-notifications.svg
Add the package providing the target to neededforbuild and Requires
ERROR: link target doesn't exist (neither in build root nor in installed system):
  /usr/share/themes/Pop-dark/gnome-shell/process-working.svg -> /usr/share/themes/Pop-dark/gnome-shell/assets/process-working.svg
Add the package providing the target to neededforbuild and Requires
ERROR: link target doesn't exist (neither in build root nor in installed system):
  /usr/share/themes/Pop-dark-slim/gnome-shell/no-events.svg -> /usr/share/themes/Pop-dark-slim/gnome-shell/assets/no-events.svg
Add the package providing the target to neededforbuild and Requires
ERROR: link target doesn't exist (neither in build root nor in installed system):
  /usr/share/themes/Pop-dark-slim/gnome-shell/no-notifications.svg -> /usr/share/themes/Pop-dark-slim/gnome-shell/assets/no-notifications.svg
Add the package providing the target to neededforbuild and Requires
ERROR: link target doesn't exist (neither in build root nor in installed system):
  /usr/share/themes/Pop-dark-slim/gnome-shell/process-working.svg -> /usr/share/themes/Pop-dark-slim/gnome-shell/assets/process-working.svg
Add the package providing the target to neededforbuild and Requires
error: Bad exit status from /var/tmp/rpm-tmp.j1a3d8 (%install)
```

And indeed all of the `assets` folders are missing and the symlinks are pointing into nirvana. Any idea where I might look at or what might cause this? I've seen that the `assets` folder should have been copied and surely `cp` would have complained if something would have failed there.

Thanks!

**Comment by isantop** [comment]
Thanks for opening an issue! What version of the theme do you have pulled? If you can get the git commit hash, that would be even better.

**Comment by brejoc** [comment]
It's the [version 3.0.0-b4](https://github.com/pop-os/gtk-theme/releases/tag/3.0.0-b4). Commit has would be this: https://github.com/pop-os/gtk-theme/commit/7ae139d94d9ba19d465edf6a2cb5a63cd45b81d8
Thanks!

**Comment by isantop** [comment]
Yep, that bug was fixed in 43b9291db1299de43d79410748c20c6c8a4ffdb8. It would be best to pull the latest code from master (commit 35b69b72b771223fa0c078856dde64e793e8d852 )

**Comment by brejoc** [comment]
@isantop Thanks! Seems to work!
